content_handler/runtime_holder.cc

```diff
@@ -570,9 +570,22 @@ void RuntimeHolder::OnEvent(mozart::InputEventPtr event,
           pointer_data.change = blink::PointerData::Change::kHover;
         break;
       case blink::PointerData::Change::kAdd:
+        if (down_pointers_.count(pointer_data.device) != 0) {
+          FXL_DLOG(ERROR) << "Received add"
+                          << " event for pointer that is currently down!";
```

Comment:
Maybe combine these two strings.  I'd also remove the ! and use a .  (Same comment below.)